src/mixins/filing-template-mixin.ts

```diff
@@ -368,7 +369,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
             : {}
         },
         nameRequest: {
-          legalName: this.getNameRequest.details['approvedName'],
```

Comment:
This was a sloppy way to work around a type issue.

Reply:
Using `(myObject as any).myProperty` is also not ideal, but lint can flag variables used as any so we could find and fix these in the future. Plus there aren't a lot of places where "as any" is used 😅